netctl/http.go

```diff
@@ -54,7 +54,10 @@ func postMap(ctx *cli.Context, url string, jsonMap map[string]interface{}) {
 
 	resp, err := client.Post(url, "application/json", bytes.NewBuffer(content))
 	if err != nil {
-		writeBody(resp, ctx)
+		if resp != nil {
```

Comment:
But really, I'm having trouble thinking of cases where we have both an error and a body, and I'm struggling. Not quite sure what I was thinking now. Maybe we should just remove it?